linodecli/configuration/helpers.py

```diff
@@ -206,86 +258,20 @@ def _config_get_with_default(
     user: str,
     field: str,
     default: Any = None,
-) -> Optional[Any]:
+) -> Any:
     """
     Gets a ConfigParser value and returns a default value if the key isn't found.
+
+    :param user: The user to get a value for.
+    :type user: str
+    :param field: The name of the field to get the value for.
+    :type field: str
+    :param default: The default value to use if a value isn't found. Defaults to None.
+    :type default: Any
+
+    :returns: The value pulled from the config or the default value.
+    :rtype: Any
     """
     return (
         config.get(user, field) if config.has_option(user, field) else default
     )
-
-
-def _handle_no_default_user(self):  # pylint: disable=too-many-branches
```

Comment:
`self` in this case referenced the CLIConfig, so I moved it back into the CLIConfig class as a method 👍